pkg/controller/nodemaintenance/check_lease_supported_test.go

```diff
@@ -0,0 +1,43 @@
+package nodemaintenance
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+    . "github.com/onsi/ginkgo/extensions/table"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+
+var _ = Describe("checkIfLeaseAPISupported", func() {
+
+	DescribeTable("check lease supported",
+		// common test function invoked by table test.
+        func(initWithLeaseAPIPackage bool, leaseSupported bool) {
```

Comment:
Start with this, and go from there...

```suggestion
	It("should enable leases when supported", func() {
		for _, c := range cases {
```

Reply:
Finishing off the code you were provided with on Monday is about an hour's worth of work.
We've been talking about [validaterbacstatus_test.go](https://github.com/kubernetes/dashboard/blob/master/src/app/backend/validation/validaterbacstatus_test.go) for 7 weeks, can we move on from this part of the review please?

Reply:
done.

Reply:
changed it yet again, an It clause within a loop doesn't work, but Context clause within a loop does work.